README.md

```diff
@@ -1 +1,55 @@
-Awesome Books 
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Project Name
```

Comment:
- [ ] Kindly remove the placeholder `Project Name` and replace it with your current project title. This makes your project look more professional.

Reply:
@brytebee , we have made the changes